### DISCUSSION TITLE
A problem with denoising.

### DISCUSSION THREAD

**Discussion by yushuv** [tbepler/topaz#89] [cryoSPARC]
Hi all, 
I have an odd problem with denosing. Not all the time I get the pictures of the denoised micrograph. It says it's done but I can't see anything even when I try to look at all of them and not just the ones in the cryosparc gui.

In the picture I added here I can see the that in the big dataset (J10) I can see them and in the small one (J14 and J16) I can't see any of them. I want to use these to see if the picking is right and since I can't see the denoise ones I can't see much if the picking is well enough for me. 

I run them the same. 

<img width="931" alt="Screenshot 2021-03-18 at 14 04 23" src="https://user-images.githubusercontent.com/79204086/111639449-2efbf900-87f3-11eb-9dd0-3f8030d8b740.png">

Thanks, 
Yehuda

**Comment by tbepler** [comment]
This looks like a cryoSPARC problem rather than a Topaz problem, per se, so you may get better help on the cryoSPARC forums (https://discuss.cryosparc.com/).

@alexjnoble Have you ever run into this issue with the cryoSPARC integration before?

**Comment by alexjnoble** [comment]
Hi Yehuda,

I'm not sure what the error is from. Could you share the job output for both working and non-working denoise jobs?

Thanks,
-Alex

**Reply by yushuv** [reply]
Hi Alex, 
Here are the two files of the outpout. 
Sorry it toke so long we had power testing over the weekend so no computers. 


Yehuda


[fail.docx](https://github.com/tbepler/topaz/files/6182631/fail.docx)
[worked.docx](https://github.com/tbepler/topaz/files/6182633/worked.docx)

**Comment by alexjnoble** [comment]
Hi Yehuda,

The job runs out of RAM. To solve this, decrease the size of the 'Shape of split micrographs' parameter.

Best,
-Alex

**Reply by yushuv** [reply]
Hi Alex, 

I lowered it until 256 and it's still only doing a part of the micrographs. Is there anything else I could try?

**Reply by alexjnoble** [reply]
Hi Yehuda,

Could you try running it on CPU instead of GPU? There's an option in Cryosparc, I think.

Thanks,
-Alex

**Reply by yushuv** [reply]
Ok on CPU it works. 
Maybe we have a deep GPU problem then we thought. 

Thanks, 
Yehuda
